lib/hutch/config.rb

```diff
@@ -13,6 +13,7 @@ def self.initialize(params={})
         mq_host: 'localhost',
         mq_port: 5672,
         mq_exchange: 'hutch',  # TODO: should this be required?
+        mq_exchange_options: {},
```

Comment:
Some unsolicited advice, I would make this default to `{ durable: true}` instead of blank.

That way, anyone who upgrades won't get any big surprises, as in, their queues aren't durable anymore.